hyperdash_cli/cli.py

```diff
@@ -281,6 +283,10 @@ def stderr_loop():
     wrapped()
 
 
+def version(args=None):
+    print("hyperdash {}".format(get_hyperdash_version()))
```

Comment:
Omit `args=None`?

Reply:
I think it's fine - emphasizes the method is connected to line 414

Reply:
Looking at the other methods here, `run` is the only one that receives `args`, so it feels weird that `run` is not the only function with `args` in the function declaration.